Source/Core/Core/IOS/USB/OH0/OH0.cpp

```diff
@@ -311,6 +314,7 @@ IPCCommandResult OH0::DeviceIOCtlV(const u64 device_id, const IOCtlVRequest& req
                           [&, this]() { return SubmitTransfer(*device, request); });
   case USB::IOCTLV_USBV0_UNKNOWN_32:
     request.DumpUnknown(GetDeviceName(), LogTypes::IOS_USB);
+    DolphinAnalytics::Instance()->ReportUnknownIOSRequest(GetDeviceName(), request);
     return GetDefaultReply(IPC_SUCCESS);
   default:
     return GetDefaultReply(IPC_EINVAL);
```

Comment:
Asked that when the code was initially written; its because there are no other known IOCTLs beyond the ones in the enum - the default case should never be hit in practise (other than in broken homebrew)